### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AtomicRobotMan0101**
reading fluidity and a few trivial grammatical fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,11 +94,10 @@ Need something more? Create an issue, we listen.
 
 ## Why did we pursue this?
 
-For many years now, **Requests** has been frozen and blocked millions of developers, left in a vegetative state
-regarding evolutions in the field.
+For many years now, **Requests** has been frozen. Being left in a vegetative state and not evolving, tihs blocked millions of developers from using more advanced features.
```
Comment (Ousret) [inline README.md:97]:
```suggestion
For many years now, **Requests** has been frozen. Being left in a vegetative state and not evolving, this blocked millions of developers from using more advanced features.
```

File: README.md [hunk 2]
```diff
@@ -61,7 +60,7 @@ Niquests officially supports Python or PyPy 3.7+.
 
 ## Supported Features & Best–Practices
 
-Niquests is ready for the demands of building robust and reliable HTTP–speaking applications, for the needs of today.
+Niquests is ready for the demands of building modern, robust and reliable HTTP–speaking applications.
```
Comment (Ousret) [inline README.md:63]: "modern" only tend to "trigger" people. Maybe choosing something else..?

**Review by Ousret — CHANGES_REQUESTED** [review]
I've got few comments on my initial reading.

File: README.md (hunk 1 above)
Comment (AtomicRobotMan0101) [reply README.md:97]: well found!

File: README.md (hunk 2 above)
Comment (AtomicRobotMan0101) [reply README.md:63]: Given the ability to use concurrency, perhaps eliminate modern entirely and use "scalable".

Comment (Ousret) [inline README.md:63]:
```suggestion
Niquests is ready for the demands of building scalable, robust and reliable HTTP–speaking applications.
```

**Review by Ousret — APPROVED** [review]

**Pull request merged**